okio/src/jvmMain/kotlin/okio/AsyncTimeout.kt

```diff
@@ -180,7 +163,7 @@ open class AsyncTimeout : Timeout() {
    * [InterruptedIOException]. If [cause] is non-null it is set as the cause of the
    * returned exception.
    */
-  protected open fun newTimeoutException(cause: IOException?): IOException {
+  open fun newTimeoutException(cause: IOException?): IOException {
```

Comment:
I love this change in its current state.

But there’s a problem. I don‘t think this is a backwards-compatible change. Subclasses might have `protected` visibility, and widening it to `public` will break those.

The fix is pretty easy, just create a binary-public trampoline?

```
@Deprecated(level = DeprecationLevel.HIDDEN)
@JvmName("-newTimeoutException")
fun dashNewTimeoutException(cause: IOException) = newTimeoutException(cause)
```

Reply:
Good catch, thanks!

I had to do something a little different as `HIDDEN` level also makes it hidden to the current module.